index.js

```diff
@@ -4,7 +4,10 @@ module.exports = Object.assign(
     validate: require('./validate'),
     is: require('./is'),
     struct: require('./struct'),
-    compose: require('./compose')
+    compose: require('./compose'),
+    maybe: require('./maybe'),
+    some: require('./some'),
+    dict: require('./dictionary'),
```

Comment:
i think these new additions should be "types", and maybe with capital letters. (at least `Dict` should be capital, but why not all of them).